packages/paste-website/src/pages/components/heading/index.mdx

```diff
@@ -0,0 +1,209 @@
+
+- Headings can have multiple levels of depth to help organize content
+- Heading types provide specific styles to the font-size, font-weight, and line-height of the text to create hierarchy that allow users to easily scan pages and content
+- Headings should be used with intention that maintains a coherent document outline
+- Don’t use heading tags to resize text. Instead use the `<Text>` utility
+- Don’t skip headline levels. Always start with `<h1>`, `<h2>`, and so on
+- There can only be one `<h1>` tag per page.
+
+#### Accessibility
+
+It is important to not skip one or more heading levels as it is common for screen readers to build an outline of the page and assist users as they navigate a page. Skipping a heading level could cause confusion as the user navigating the page may not know why a heading is missing.
+
+- Heading text should meet AA requirements (4.5:1) for color contrast from itself and the background color
+- Headings should be used thoughtfully, with clear hierarchies for content and subsections
+
+## Examples
+
+<LivePreview scope={{Heading}} language="jsx">
+  {`<div>
+  <Heading as="h1" variant="headingStyle60">
+    Heading One
+  </Heading>
+
+  <Heading as="h2" variant="headingStyle50">
+    Heading Two
+  </Heading>
+
+  <Heading as="h3" variant="headingStyle40">
+    Heading Three
+  </Heading>
+
+  <Heading as="h4" variant="headingStyle30">
+    Heading Four
+  </Heading>
+
+  <Heading as="h5" variant="headingStyle20">
+    Heading Five
+  </Heading>
+
+  <Heading as="h6" variant="headingStyle10">
+    Heading Six
+  </Heading>
+</div>`}
+</LivePreview>
+
+#### Heading One
+
+<LivePreview scope={{Heading}} language="jsx">
+  {`<Heading as="h1" variant="headingStyle60">
+  Heading One
+</Heading>`}
+</LivePreview>
+
+#### Heading Two
+
+<LivePreview scope={{Heading}} language="jsx">
+  {`<Heading as="h2" variant="headingStyle50">
+  Heading Two
+</Heading>`}
+</LivePreview>
+
+#### Heading Three
+
+<LivePreview scope={{Heading}} language="jsx">
+  {`<Heading as="h3" variant="headingStyle40">
+  Heading Three
+</Heading>`}
+</LivePreview>
+
+#### Heading Four
+
+<LivePreview scope={{Heading}} language="jsx">
+  {`<Heading as="h4" variant="headingStyle30">
+  Heading Four
+</Heading>`}
+</LivePreview>
+
+#### Heading Five
+
+<LivePreview scope={{Heading}} language="jsx">
+  {`<Heading as="h5" variant="headingStyle20">
+  Heading Five
+</Heading>`}
+</LivePreview>
+
+#### Heading Six
+
+<LivePreview scope={{Heading}} language="jsx">
+  {`<Heading as="h6" variant="headingStyle10">
+  Heading Six
+</Heading>`}
+</LivePreview>
+
+## Composition Notes
+
+### When to use Heading
+
+Headings should be used for page titles or subsection title. Headings are broken into six levels, where h1 is the highest level and h6 is the lowest. Avoid skipping heading levels. Use the next lowest heading level to maintain a coherent document outline.
```

Comment:
Semantic nit: lets keep the terms titles and headings separate. This can lead to important confusion. I recently saw an accessibility audit calling out conformance to a WCAG item regarding page titles, but confused the remediation to be heading level 1 elements on the page. Whereas in fact the WCAG item is after an actual page title

```
<head>
  <title>This is the title of the page</title>
</head>
```

These are Headings and content sub headings. Probably need to change the intro sentence too

Reply:
I did a little word smithing to remove mention of page titles.